tests/test_integration.py

```diff
@@ -228,15 +248,15 @@ def test_noise_model_qasm_simulator(self, monkeypatch):
             m.setattr(
                 qiskit_aer.AerSimulator, "set_options", lambda *args, **kwargs: cache.append(kwargs)
             )
-            dev = qml.device("qiskit.aer", wires=2, noise_model="test value")
+            qml.device("qiskit.aer", wires=2, noise_model="test value")
```

Comment:
No need for "dev"?

Reply:
Not really, especially under the `with` context; we simply are checking whether this statement gets executed with the provided context condition or not. Had this been not under the context we would have needed assignment.